ui/src/pages/AllStories/reducer.ts

```diff
@@ -1,35 +1,57 @@
-import { StoryView } from "../AllStories";
+import { StoryView } from "../AllStories/index";
 
 export interface State {
   tabValue: number;
+  search: string;
   visibleState: StoryView[];
   visibleTableState: StoryView[];
```

Comment:
Can you merge in `main` or rebase? There's a couple of state changes including the old search that's making the diff kind of confusing. i.e. this visibleState should be a number[]